tests/integration/_cases/debug_files/debug_files-help.trycmd

```diff
@@ -21,6 +21,7 @@ OPTIONS:
 SUBCOMMANDS:
     bundle-sources    Create a source bundle for a given debug information file
     check             Check the debug info file at a given path.
+    dump-sources      Print source files linked by the given debug info file.
```

Comment:
alternative name I was thinkig of: `show-sources`

Reply:
I'd go with `print-sources`. `dump` is too easy to mistype with `dumb` and can be annoying 😅

Reply:
👍